docs/en/tutorials/getting-started.rst

```diff
@@ -170,15 +170,10 @@ factory method.
 Generating the Database Schema
 ------------------------------
 
```

Comment:
The statement "Now that we have defined the Metadata mappings" is not accurate. At this point, we haven't defined any metadata mappings yet.